### DISCUSSION TITLE
ERROR: for web  Get "https://ghcr.io/v2/" Install fails on Umbrel

### DISCUSSION THREAD

**Discussion by sovereignalmida** [advplyr/audiobookshelf#2280] [Q&A]
I cannot install AudioBookShelf on Umbrel. I have dome multiple clean installs of Umbrel. installed AudioBookShelf first etc.  I always get the same error:
ERROR: for web  Get "https://ghcr.io/v2/": dial tcp: lookup ghcr.io on 192.168.1.254:53: no such host
Get "https://ghcr.io/v2/": dial tcp: lookup ghcr.io on 192.168.1.254:53: no such host

I CAN however install AudioBookShelf successfully using Portainer on Umbrel pulling the image from docker.io  This leads me to suspect there is possibly something wrong with the image or where it is being pulled form for Umbrel? my guess only.

Can anyone help resolve or shed some light on this?  I prefer to have AudioBookShelf fully managed under Umbrel  and not in a Portainer container unless I have to.

**Comment by advplyr** [comment]
I've never used Umbrel and don't know any of the maintainers for the Umbrel app. You will probably have better luck asking in Umbrel forums

**Reply by sovereignalmida** [reply]
This who submitted the app on apps.umbrel.com  fyi only.   cheers and thanks for a great app BTW  - Audiobookshelf is very good!

https://github.com/ceramicwhite
